### PR TITLE
Remove ignoreFolders option from FileSize rule

### LINKED ISSUE #971: src directory is not found in file path <...>/build.gradle.kts
## Describe the bug
This message is true, but we shouldn't log it. We have some special handling of gradle scripts, and it should include legal absence of src directory

## Environment information
* diktat version: 1.0.0-rc.2
* build tool (maven/gradle): gradle

### PR DESCRIPTION
**Pull request by petertrr**
### What's done:
* Removed code
* Fixed docs

This pull request closes #971

### CODE REVIEW

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter1/PackageNaming.kt [hunk 1]
```diff
@@ -276,7 +276,7 @@ class PackageNaming(configRules: List<RulesConfig>) : DiktatRule(
         /**
          * Directory which is considered the start of sources file tree
          */
-        const val PACKAGE_PATH_ANCHOR = "src"
+        const val PACKAGE_PATH_ANCHOR = SRC_DIRECTORY_NAME
```
Comment (kentr0w) [inline diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter1/PackageNaming.kt:279]: Maybe also move `SRC_DIRECTORY_NAME` into `FileUtils.kt` file?

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/FileSize.kt [hunk 2]
```diff
@@ -58,18 +51,9 @@ class FileSize(configRules: List<RulesConfig>) : DiktatRule(
          * Maximum allowed number of lines in a file
          */
         val maxSize = config["maxSize"]?.toLongOrNull() ?: MAX_SIZE
-
-        /**
-         * List of folders, files from which are ignored during the check. For example, for tests.
-         */
-        val ignoreFolders = config["ignoreFolders"]?.replace("\\s+".toRegex(), "")?.split(IGNORE_FOLDERS_SEPARATOR) ?: ignoreFolder
```
Comment (kentr0w) [inline diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/FileSize.kt:65]: Why now we don't have this parameter?

**Review by kentr0w — CHANGES_REQUESTED** [review]

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter1/PackageNaming.kt (hunk 1 above)
Comment (petertrr) [reply diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter1/PackageNaming.kt:279]: But it's already in there

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/FileSize.kt (hunk 2 above)
Comment (petertrr) [reply diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/FileSize.kt:65]: We have decided that we no longer need it. It was originally introduced at the time when we didn't have exclusion mechanism, but now we can add excludes into plugin config or add file-level suppress for any rule, so special setting for file size is no longer required

File: diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter1/PackageNaming.kt (hunk 1 above)
Comment (kentr0w) [reply diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter1/PackageNaming.kt:279]: Yes, confused)

**Review by kentr0w — APPROVED** [review]
lgtm

**Pull request merged**